libcrux-kem/Cargo.toml

```diff
@@ -25,5 +25,5 @@ pre-verification = ["libcrux-ml-kem/pre-verification"]
 
 [dev-dependencies]
 libcrux-kem = { version = "0.0.2-alpha.1", path = "./", features = ["tests"] }
-libcrux = { version = "0.0.2-alpha.1", path = "../", features = ["rand"] }
+rand_core = { version = "0.6" }
```

Comment:
No need for this if you use `rand::rngs::OsRng` instead.

Reply:
You' right, thanks! :+1: